src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -190,6 +249,13 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
           <View style={styles.headerText}>
             <AppHeader mode={config.appMode} />
           </View>
+          {hasMultiInputSelection() && (
+            <InputSelection
+              selectionArray={getSelectionArray()}
```

Comment:
Similarly for here, looks like we construct the array twice for both `hasMultiInputSelection` and `getSelectionArray`, useMemo should help

Reply:
have change to `useMemo`